index.md

```diff
@@ -34,11 +38,21 @@ Every large software development project relies on it,
 and most programmers use it for their small jobs as well.
 And it isn't just for software:
 books (like this one),
-papers,
+papers, theses, blogs,
 small data sets,
 and anything that changes over time or needs to be shared
 can and should be stored in a version control system.
 
+> ## Warning {.callout}
+> 
+> Git works best with plain text files that have regular line breaks.
```

Comment:
On 2015-02-07 4:50 PM, Raniere Silva wrote:
>
> In index.md 
> <https://github.com/swcarpentry/git-novice/pull/36#discussion_r24293287>:
>
> >  small data sets,
> >  and anything that changes over time or needs to be shared
> >  can and should be stored in a version control system.
> >
> > +> ## Warning {.callout}
> > +>
> > +> Git works best with plain text files that have regular line breaks.
>
> What about
>
> |Git works with **any** file at your computer
> and better with plain text files
> like programming source code and marked up text.
> |

This shouldn't be put in the introduction because the reasons won't be 
apparent yet - instead, please put this in `03-conflict.md` where we 
discuss diff/merge (because inability to do that with non-text files is 
why version control is awkward for them).

-- 
Dr. Greg Wilson    | gvwilson@software-carpentry.org
Software Carpentry | http://software-carpentry.org